src/zenml/client.py

```diff
@@ -1205,6 +1209,234 @@ def register_pipeline(
         )
         raise AlreadyExistsException(error_msg)
 
+    def export_pipeline_runs(self, filename: str) -> None:
+        """Export all pipeline runs to a YAML file.
+
+        Args:
+            filename: The filename to export the pipeline runs to.
+
+        Raises:
+            RuntimeError: If no pipeline runs exist.
+        """
+        import json
+
+        from zenml.utils.yaml_utils import write_yaml
+
+        pipeline_runs = self.zen_store.list_runs(
+            project_name_or_id=self.active_project.id
+        )
+        if not pipeline_runs:
+            raise RuntimeError("No pipeline runs found.")
```

Comment:
This could be handled more gracefully with a simple warning message that nothing got exported IMO

Reply:
I agree, makes sense.